src/layouts/MultiDhoLayout.vue

```diff
@@ -447,7 +447,7 @@ q-layout(:style="{ 'min-height': 'inherit' }" :view="'lHr Lpr lFr'" ref="layout"
           q-btn.q-px-xl.rounded-border.text-bold.full-width(:to="{ name: 'configuration', query: { tab: 'PLAN' } }" color="white" text-color="negative" :label="$t('layouts.multidholayout.renewMyCurrentPlan')" no-caps rounded unelevated)
   //- Because iOS z-index doesn`t work
   router-view(v-if="$router.currentRoute.name === 'proposal-create' && $q.screen.lt.md")
-  q-header.bg-white(v-if="$q.screen.lt.lg && $route.name !== 'proposal-detail'")
+  q-header.bg-white(v-if="$q.screen.lt.lg && $route.name !== 'proposal-detail' && $route.name !== 'create-your-dao'")
```

Comment:
@evgenibir `create-your-dao` should be constant

Reply:
@Alex-IS fixed!